### ISSUE TITLE
Corsair Vengeance RGB (non Pro) and ASUS GPUs

### ISSUE THREAD

**Issue by xLMDCx** [augustinbegue/icue-ambilight#39]
My Asus Strix RTX 2060 is not getting controlled by the Software, even if it shows up in the Devices List and my 2 Vengeance RGB (non Pro) dont even show up.

Anyway, the Node Pros are working perfectly fine for me. I really love your Software, its well made and pretty fast (Asus´s Ambilight function isnt as fast as yours)

![iCUE 29 08 2021 20_12_54](https://user-images.githubusercontent.com/76822327/131261069-df042a43-7967-4769-9dd8-9c875c726495.png)
![icue-ambilight 29 08 2021 20_13_08](https://user-images.githubusercontent.com/76822327/131261072-5b86bdf4-d285-4144-84de-e05163843123.png)

**Comment by augustinbegue** [comment]
Hi!
To begin, sorry for the extreme inactivity, school is giving me a hard time to find some free time 🙃...

Thanks for opening this issue. This is probably caused by the outdated version of the icue sdk the software is using, so I'll update it.
Since I do not own any of these devices, I won't be able to test if it solved the issue, so please get back in touch to see if the update fixed the issue (when it will be done).

**Comment by xLMDCx** [comment]
So the Problem with my Strix Card is Fixed now! Thank You very much. But Sadly the Ram still dont get recognized by your Software.

**Comment by augustinbegue** [comment]
That's good news!
For the ram, could you check if there is an option to "enable full software control" ? I don't know if it still exists in the last versions of icue but that may be why the device is not seen by the sdk.

**Comment by xLMDCx** [comment]
Yes there is and it's Activated, otherwise my Node Pros and GPU won't work. Today I've installed a new Update of iCue, maybe that changed something important...

**Comment by xLMDCx** [comment]
I have checked it again to be sure, the Ram don't show up in the Settings Tab for Software Integration. I think they can't really be controlled by a Plugin,but if you manage to get them working it would be amazing... If you have some prereleases or something, you can just text me here. Merry Christmas btw!

**Comment by augustinbegue** [comment]
> I think they can't really be controlled by a Plugin

I'll try to do more research to see if it really the case, because it seems odd from Corsair to do that. I'll keep you updated if I find a solution.
Thanks and Merry Christmas too !
